Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -78,9 +78,11 @@ void CSSPageRule::setSelectorText(const String& selectorText)
 
 String CSSPageRule::cssText() const
 {
+    auto selector = selectorText();
+    auto optionalSpace = selector.isEmpty() ? ""_s : " "_s;
     if (auto declarations = m_pageRule->properties().asText(); !declarations.isEmpty())
-        return makeString(selectorText(), " { "_s, declarations, " }"_s);
-    return makeString(selectorText(), " { }"_s);
+        return makeString("@page "_s, selector, " { "_s, declarations, " }"_s);
```

Comment:
```suggestion
        return makeString("@page"_s, optionalSpace, selector, " { "_s, declarations, " }"_s);
```

Reply:
It was not progressing the failing test, so I didn't done the change here. Although now I have done, let's see from EWS, if it do any progression or regression. :-)

Reply:
If there is no progression then that tells us we have insufficient test coverage! Almost certainly not that the change is unneeded.

Reply:
When I did earlier without `space`, it did regress few things. So I think we do have test coverage but I was keeping it `@page `, so it was not reflecting. I remember one of my EWS showing failure when I removed space from first return.

Reply:
The two consecutive spaces would be a bug. We should do it correctly and make sure the tests expect the correct behavior.

Reply:
@Ahmad-S792 is this tested now?

Reply:
@annevk - thanks for your guidance on Slack 1-1, I have extedned - `css/cssom/cssom-pagerule.html` to test `cssText` path as well. Appreciate if you can review. :-)